### PR TITLE
Adding dish to menu

### PR DESCRIPTION
**Pull request by jocontreras**
PT link: https://www.pivotaltracker.com/story/show/142582451

**Changes proposed in this PR:**
Adds feature for adding dish to menu
Adding happy path 
Adding sad path

**What I have learned working on this feature:**
I improved understand on adding features and use, flash message and formatting simple form
I learned adding error message 

screenshot from 2017-04-05 17-12-39

### CODE REVIEW

File: app/controllers/dish_controller.rb [hunk 1]
```diff
@@ -3,4 +3,30 @@ class DishController < ApplicationController
...
+  end
+
+  def create
+    @menu = Menu.find(params[:menu_id])
+    @dish = Dish.new(dish_params)
+    if @dish.save
+      flash[:notice] = "A new dish has been added to #{@menu.name} menu"
+      redirect_to restaurant_menu_path(@menu.restaurant, @menu)
+    else
+      flash[:notice] = 'Please make sure to fill in all the fields'
```
Comment (tochman) [inline app/controllers/dish_controller.rb]: A bit to generic for my taste. I would like to see the actual errors you get with `@dish.errors`

File: app/views/dish/new.html.haml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+.row
+  .medium-8.medium-centered
+    %h2 Add new dish to '#{@menu.name}' menu
+    - if @dish.errors.any?
+      .callout.small.alert
+        %ul
+        - @dish.errors.full_messages.each do |error|
+          %li
+            = error
```
Comment (tochman) [inline app/views/dish/new.html.haml]: ...as I see that you use here... Okay. Still if feels a bit redundant to have both a flash and the error messages displayed. Not a blocker though.

File: features/add_dishes.feature [hunk 3]
```diff
@@ -1,24 +1,35 @@
-Feature: Add dishes to menu
-  As a user
-  In order to decide from various dishes from a restaurant
-  I would like to see many dishes in the restaurant´s menu
+Feature: Add dishes to restaurant
+  As a restaurant owner
+  In order to manage my menu
+  I would like to be able to add dishes on my menu
```
Comment (tochman) [inline features/add_dishes.feature:4]: Okay, so I'm NOT found of changing the user story on existing features. I would prefer you create a new feature file. The question is why was the feature written the way it was before. Seems weird to me. Anyway seems like the original story was about displaying dishes and your is about adding them. Why was it named `add_dishes.feature` originally. (Please check git blame and ask for explanation)

**Review by tochman — APPROVED** [review]

File: features/add_dishes.feature (hunk 3 above)
Comment (sigu) [reply features/add_dishes.feature:4]: We discussed with @casperjureskog earlier this morning and we realised it was grammatical oversight on there side. The editing of this section was done together with him. Git blame is blaming me and I dont remember commiting this file :thinking:

Comment (tochman) [reply features/add_dishes.feature:4]: Alright.

**Pull request merged**